src/main/java/it/gov/pagopa/pu/sil/endpoint/PuForOrganizationPaymentsEndpoint.java

```diff
@@ -316,4 +321,47 @@ public PaaSILRegistraPagamentoRisposta paaSILRegistraPagamento(
       PaaSILRegistraPagamentoRisposta::setFault
     );
   }
+
+  @PayloadRoot(namespace = NAMESPACE_URI, localPart = "paaSILPrenotaExportFlusso")
+  @ResponsePayload
+  public PaaSILPrenotaExportFlussoRisposta paaSILPrenotaExportFlusso(
+    @RequestPayload PaaSILPrenotaExportFlusso request,
+    @SoapHeader("{http://www.regione.veneto.it/pagamenti/ente/ppthead}intestazionePPT") SoapHeaderElement header) {
+    UserInfo userInfo = SecurityUtils.getLoggedUser();
+    String accessToken = SecurityUtils.getAccessToken();
+    String orgIpaCode = SoapUtils.getOrganizationIpaCodeFromHeader(header,
+      IntestazionePPT.class,
+      IntestazionePPT::getCodIpaEnte,
+      "paaSILPrenotaExportFlusso");
+
+    return registryLogger.execute(
+      AuthorizationService.getOrgFiscalCodeFromUserInfo(userInfo, orgIpaCode),
+      RegistrySilEventType.paaSILPrenotaExportFlusso,
+      null,
+      request,
+      userInfo,
+      null,
+      () -> {
+        Long result = paaSILPrenotaExportFlussoService.paaSILPrenotaExportFlusso(
+          userInfo,
+          accessToken,
+          orgIpaCode,
+          request
+        );
+        PaaSILPrenotaExportFlussoRisposta response = new PaaSILPrenotaExportFlussoRisposta();
+        response.setRequestToken(String.valueOf(result));
+        return Triple.of(response, null, SilOutcome.OK);
+      },
+      FaultUtils.unauthorizedOrSystemExceptionHandler(
```

Comment:
The invoked API could return error for the following use cases:
* fileVersion non handled
* range limit exceeded

There are some particular ERROR CODE to return?